log/log_test.go

```diff
@@ -1 +1,160 @@
 package log
+
+import (
+	"bytes"
+	"log"
+	"strings"
+	"testing"
+)
+
+func TestOutputf(t *testing.T) {
+	const string1 = "asdf123"
+	buf := new(bytes.Buffer)
+
+	log.SetOutput(buf)
+	outputf(LevelDebug, string1, nil)
+
+	// test 1: outputf correctly prints string
+	if !strings.Contains(buf.String(), string1) {
+		t.Fail()
+	}
+
+	(*bytes.Buffer).Reset(buf)
+	outputf(LevelDebug-1, string1, nil)
```

Comment:
It turns out that there is no Level constant lower than LevelDebug. So it seems like there is no need to include that second test... Thanks! I will fix.